go/obscuronode/enclave/db/storage.go

```diff
@@ -4,14 +4,15 @@ import (
 	"bytes"
 	"fmt"
 
+	rawdb2 "github.com/obscuronet/obscuro-playground/go/obscuronode/enclave/db/rawdb"
```

Comment:
Could we give a more useful name for this one too please? Call one of them `gethrawdb` or `obsrawdb` or something perhaps.